apache/metron
# METRON-1720: Better error messages when there are no results or wireshark is not installed

## Contributor Comments
This PR adds error handling when pdml results are requested after a pcap job has finished.  I also added a small fix to a pcap-panel-component.ts test where the job status wasn't properly set.

### Testing
This has been tested in full dev:

1. Navigate to the Pcap tab in the Alerts UI
2. Submit a Pcap query with a filter you know won't match any results
3. After the job finishes you should see an error message that says "No results found"
4. Uninstall wireshark in full dev:  `yum -y remove wireshark`
5. Run a query that will produce results
6. You should get an error message that includes a message about tshark not being found

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  


In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:


### For code changes:
  ```
  mvn -q clean integration-test install && dev-utilities/build-utils/verify_licenses.sh 
  ```


### For documentation related changes:

  ```
  cd site-book
  mvn site
  ```

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.
